tests/e2e/test_rosacli_cluster.go

```diff
@@ -3067,3 +3063,105 @@ var _ = Describe("Sts cluster creation supplemental testing",
 				Expect(clusterID).ToNot(BeNil())
 			})
 	})
+
+var _ = Describe("User can set availability zones to create rosa multi-az STS cluster supplemental testing",
```

Comment:
Why a new Describe?  Can this be put to other Describe? I believe there are other day1 supplemental cases existing

Reply:
Updated using the existed day1 supplemental Describe.